boards/arm/actinius_icarus/doc/index.rst

```diff
@@ -0,0 +1,127 @@
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter with EasyDMA)`
+* :abbr:`WDT (Watchdog Timer)`
+* :abbr:`IDAU (Implementation Defined Attribution Unit)`
+
+.. figure:: img/Icarus_pinouts.png
+     :width: 450px
+     :align: center
+     :alt: Icarus IoT Dev Board w/ Pinouts
+
+     Icarus IoT Dev Board w/ Pinouts
+
+.. figure:: img/Icarus_internal_pinouts.png
+     :width: 200px
+     :align: center
+     :alt: Icarus IoT Dev Board
+
+     Internal Pinouts
+
+Hardware
+********
+
+The detailed information about the on-board hardware can be found at (`Icarus Product Website`_)
+
+Supported Features
+==================
+
+The actinius_icarus board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| SPU       | on-chip    | system protection    |
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| ACCEL     | st         | lis2dh               |
++-----------+------------+----------------------+
+
+
+Security components
+===================
+
+- Implementation Defined Attribution Unit (`IDAU`_).  The IDAU is implemented
+  with the System Protection Unit and is used to define secure and non-secure
+  memory maps.  By default, all of the memory space  (Flash, SRAM, and
+  peripheral address space) is defined to be secure accessible only.
+- Secure boot.
+
+Building Secure/Non-Secure Zephyr applications
+==============================================
+
+The process requires the following steps:
+
+1. Build the Secure Zephyr application using ``-DBOARD=actinius_icarus`` and
+   ``CONFIG_TRUSTED_EXECUTION_SECURE=y`` in the the application project configuration file.
+2. Build the Non-Secure Zephyr application using ``-DBOARD=actinius_icarus_ns``.
+3. Merge the two binaries together.
```

Comment:
Aren't the files merged automatically? If I build hello_world sample with `actinios_icarus_ns` board selected, I get `build/zephyr/merged.hex` generated by the build system automatically.